source/dyaml/parser.d

```diff
@@ -432,7 +432,7 @@ final class Parser
             uint tagHandleEnd;
 
             //Get anchor/tag if detected. Return false otherwise.
-            bool get(const TokenID id, const Flag!"first" first, ref string target)
+            bool get(const TokenID id, const Flag!"first" first, ref string target) @trusted
```

Comment:
That's quite a wide `@trusted`  - can't you use a local @trusted lambda?

Reply:
I probably should have .idup'd instead to be consistent...